book/website/reproducible-research/rdm/rdm-elns.md

```diff
@@ -0,0 +1,173 @@
+Changing providers does not impact on your ability to access your past notebooks or to continue operating with the same workflow in future ones.
+This is not necessarily true of ELNs.
+
+Few active measures are needed to maintain the data in your paper notebooks, they are vulnerable as they exist in only one copy but as long as they are kept in a cool, dry and dark spot for example a fireproof safe they will likely last decades.
+Electronic data requires much more active upkeep
+Electronic data can *if manged properly* be more resilient to physical threats such as fire and flooding as it can frequently be backed up in multiple locations.
+A hard drive or even a solid state drive however cannot be left in a draw for a decade or more and have a high likelihood of working without some amount of bitrot or compatability issue when plugged back in.
+
+Lab notebooks perform an archival function and proprietary formats are antithetical to this as they assume the institution which can act as a gatekeeper to the use of the proprietary format will outlive the need to archive the material.
+When choosing an archival format one seeks to maximize the likelihood that one can recover the relevant information from that format.
+Using a proprietary solution for archival purposes is talking a needless risk with the future of your data.
+Your data's fate can become tied to that of the company, or project, that develops and operates the software that you use to store your lab notes.
+
+When looking for any critical piece of software for long term use the first questions to ask are:
+- Is there an Libre / open-source solution to this problem?
+- If it is a web app and/or has a server: Could I host my own fully featured instance, should I need to?
+- Is there a large/active community using the project, does it have institutional backing of some kind? 
+  This might take the form of a company which sells service contracts, or offers paid hosting ideally with feature parity with a self-hosted option. 
+  Or perhaps a foundation or other non-profit/academic organisation with robust funding.
+- What are the data export options and formats?
+
+Open solutions provide you with the assurance that if you do the appropriate preparatory work you should be able to access all of your data in it's native form by running the ELN application in a virutual machine or similar reproducible computational environment in the future should you need to.
+Even if the tools are no longer maintained and in a state that can be used in production.
+They can still be used to read the data and interact with it in (mostly) the same way.
+The data will also likely be stored in an open format from which it can relatively easily be extracted and ported to a new format.
+This may still be a lot of work but it is much more attainable than reverse engeering a proprietary format for which there is no documentation or worse still dealing with encrypted files.
```

Comment:
```suggestion
This may still be a lot of work but it is much more attainable than reverse engineering a proprietary format for which there is no documentation or worse - dealing with encrypted files.
```